chart/values.yaml

```diff
@@ -0,0 +1,59 @@
+# -- Number of replicas for the deployment
+replicaCount: 1
+
+common:
+  # -- Labels to add to all deployed resources
+  labels: {}
+  # -- Namespace to deploy the resources
+  namespace: "yunikorn"
+
+image:
+  # -- Docker registry
+  registry: "docker.io"
+  # -- Docker image repository
+  repository: "gresearch/yunikorn-history-server"
+  # -- Docker image tag
+  tag: "latest"
+  # -- Docker image pull policy
+  pullPolicy: "IfNotPresent"
+
+service:
+  # -- Service type
+  type: LoadBalancer
+  # -- Service port
+  port: 8989
+  # -- Service target port
+  targetPort: 8989
+
+config:
+  # -- Yunikorn scheduler host
+  yunikornHost: ""
```

Comment:
this can be grouped into a `yunicorn` block:
```
yunicorn:
  host:
```

Reply:
updated